### DISCUSSION TITLE
eicons

### DISCUSSION THREAD

**Discussion by thisura1994** [elementor/elementor#14230] [Developers Community]
Any suggestion how to get rid of this? For the nav menu mobile hamburger is it possible to replace the icon with my own SVG?

<img width="782" alt="Screenshot 2021-03-15 at 1 17 40 AM" src="https://user-images.githubusercontent.com/76260063/111082059-4a81ae00-852c-11eb-8f0c-d8f1e0f5c68f.png">

**Comment by alexcumbers** [comment]
This is a good question and would save about 80K per page, when likely very few websites are using very few of the icons. A user definable resource for hamburger menu / user specific resources would be great option to replace eIcons. please comment.

**Reply by Thisurajay** [reply]
Yes you’re absolutely correct. Agreed!

**Reply by dasignlady** [reply]
+1 for custom 'burger menu icon!!!!!

**Comment by alexcumbers** [comment]
The new 3.2 beta release improves the eIcon situation, but the ability to specify your own (SVG's), especially for the front end is needed, to avoid  unnecessary payloads.
